third_party/docfx/templates/devsite/UniversalReference.common.js

```diff
@@ -23,6 +23,12 @@ exports.transform = function (model) {
         if (model.children) groupChildren(model, namespaceCategory);
         model[getTypePropertyName(model.type)] = true;
         break;
+      // Handling subPackages differently for Python
+      case 'subpackage':
+        model.isSubpackage = true;

```

Comment:
Should it be `isSubPackage` (capital P)?

Reply:
done.